subsys/bluetooth/services/Kconfig.ets

```diff
@@ -0,0 +1,179 @@
+# Bluetooth GATT Elapsed Time Service
+
+# Copyright (c) 2025 Dipak Shetty
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BT_ETS
+	bool "GATT Elapsed Time service"
```

Comment:
The PR itself looks generally very well done, however I think it'd be important to flag the new feature as experimental, so that we can fine-tune the API in subsequent releases and users wont have an expectation of immediate API stability. So the above would become something like:
```
menuconfig BT_ETS
	bool "GATT Elapsed Time service [EXPERIMENTAL]"
	select EXPERIMENTAL
```